src/sidebar/components/hypothesis-app.js

```diff
@@ -175,16 +176,19 @@ function HypothesisApp({
     <div
       className={classnames('hypothesis-app', 'js-thread-list-scroll-root', {
         'theme-clean': isThemeClean,
+        'hypothesis-app--notebook': route === 'notebook',
```

Comment:
This is temporary, as is the conditional for `TopBar` below: `HypothesisApp` needs some refactor to not assume it will be rendering content within a sidebar, with a `TopBar` and associated layout affordances.